src/Command/Fun/GiphyCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Eve\Command\Fun;
+
+use Eve\Message;
+use Eve\Command\ClientCommand;
+
+use Eve\Command\GiphyClient;
+
+final class GiphyCommand extends ClientCommand
+{
+    /**
+     * @param Message $message
+     *
+     * @return bool
+     */
+    public function canHandle(Message $message): bool
+    {
+        return preg_match('/giphy .+/', $message->text());
+    }
+
+    /**
+     * @param Message $message
+     */
+    public function handle(Message $message)
+    {
+        $client = new GiphyClient();

```

Comment:
Could we utilise the fact that the parent class is a ClientCommand and inject a Giphy client instead of instantiating one here?